### DISCUSSION TITLE
Axis specific seamless

### DISCUSSION THREAD

**Discussion by ax53** [carson-katri/dream-textures#329] [Ideas]
First of all this addon is one of my all time favourites and I have already spent many evenings generating art. So I would like to say thank you.

Onto the suggestion. I don't know anything about how the AI stuff works but it would be cool if the seamless mode could be set to only be seamless in a specified axis, x or y.

**Comment by carson-katri** [comment]
Awesome, I’m glad you like the addon! I have actually opened a PR to add this exact feature to InvokeAI already, so if/when that is merged it will make it into Dream Textures: https://github.com/invoke-ai/InvokeAI/pull/1132

**Reply by ax53** [reply]
That sounds amazing. Looking forward to it!
